tests/testthat/TestS4/DESCRIPTION

```diff
@@ -0,0 +1,9 @@
+Package: TestS4
+Title: What the Package Does (one line, title case)
+Version: 0.0.0.9000
+Authors@R: person("Jim", "Hester", email = "james.f.hester@gmail.com", role = c("aut", "cre"))
+Description: What the package does (one paragraph).
+Depends: R (>= 3.3.0)
```

Comment:
No, `devtools::create()` just puts the current R version in there by default, definitely can use prior versions.